plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -84,20 +83,133 @@
     private CassandraServer server;
     private CassandraSession session;
 
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        switch (connectorBehavior) {
+            case SUPPORTS_CREATE_SCHEMA:
+                return false;
+
+            case SUPPORTS_CREATE_VIEW:
+                return false;
+
+            case SUPPORTS_RENAME_TABLE:
+            case SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS:
```

Comment:
nit: We can remove `SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS` since `SUPPORTS_RENAME_TABLE` is the parent. TestCassandraLatestConnectorSmokeTest is also.

Reply:
Good point. I didn't know before there exists this possibility. Thanks.